diff -up liblxqt-0.15.0/cmake/lxqt-config.cmake.in.1~ liblxqt-0.15.0/cmake/lxqt-config.cmake.in
--- liblxqt-0.15.0/cmake/lxqt-config.cmake.in.1~	2020-04-26 18:32:01.951200548 +0200
+++ liblxqt-0.15.0/cmake/lxqt-config.cmake.in	2020-04-26 18:36:02.261940875 +0200
@@ -11,7 +11,7 @@ include(CMakeFindDependencyMacro)
 find_dependency(Qt5Widgets @QT_MINIMUM_VERSION@)
 find_dependency(Qt5DBus @QT_MINIMUM_VERSION@)
 find_dependency(Qt5LinguistTools @QT_MINIMUM_VERSION@)
-find_dependency(Qt5Xdg @QTXDG_MINIMUM_VERSION@)
+find_dependency(qt5xdg @QTXDG_MINIMUM_VERSION@)
 find_dependency(KF5WindowSystem)
 find_dependency(lxqt-build-tools @LXQTBT_MINIMUM_VERSION@)
 if (NOT APPLE)
